src/contracts/markets/Abstract4626MarketWrapper.sol

```diff
@@ -143,6 +148,19 @@ contract Abstract4626MarketWrapper is Initializable, Ownable {
         revert("Not implemented");
     }
 
+    /// @notice Claim tokens from the Merkle Distributor
+    /// @param tokens The addresses of the tokens to claim.
+    /// @param amounts The amounts of the tokens to claim.
+    /// @param proofs The Merkle proofs for the claims.
+    function merkleClaim(address[] calldata tokens, uint256[] calldata amounts, bytes32[][] calldata proofs) external {
+        require(msg.sender == harvester, "Only harvester can collect");
```

Comment:
Does this need to be limited to just the Harvester? If anyone can claim the tokens, they just end up in the wrapper contract, yes?